### ISSUE TITLE
ENABLE_KEEP_MEM_NAME

### ISSUE THREAD

**Issue by M7OCM** [OneOfEleven/uv-k5-firmware-custom#22]
This enables the user to change a mem ch freq but retain the name? If so,  its stopped working as of commit before last. I'm wondering if that got fixed with latest ch tidy commit??  ;-)

**Comment by OneOfEleven** [comment]
Yes it was when I added the new channel name edit menu item, but I've fixed it now so it keeps the channel name when saving a channel.
